oximeter/oximeter/src/types.rs

```diff
@@ -231,6 +266,51 @@ pub enum DatumType {
     HistogramF64,
 }
 
+#[derive(Clone, Copy, Debug, Serialize, Deserialize)]
+pub enum DbDatumType {
```

Comment:
My intuition is that this type and `DbFieldType` should be in `oximeter/db/src/model.rs`, but I might be missing some reason that can't work.